src/shared/ui/PluginInfo.tsx

```diff
@@ -14,18 +14,24 @@ const { buildInfo: grafanaBuildInfo } = config;
 function InfoMenuHeader() {
   const styles = useStyles2(getStyles);
 
-  const {
-    meta: {
-      info: { version, updated },
-    },
-  } = usePluginContext();
+  const context = usePluginContext();
 
-  return (
-    <div className={styles.menuHeader}>
-      <h5>🔥 Explore profiles v{version}</h5>
-      <div className={styles.subTitle}>Last update: {updated}</div>
-    </div>
-  );
+  if (context) {
+    const {
+      meta: {
+        info: { version, updated },
+      },
+    } = context;
+
+    return (
```

Comment:
Good catch.
Instead of removing _entirely_ the header, why not gracefully degrade?
At least by displaying a question mark or "information not available", so it wouldn't be perceived as a bug from the user perspective.

Reply:
Good idea! How about [this](https://github.com/grafana/explore-profiles/pull/287/commits/0b3f2e1b337e42155c2c650e95383d575b32edb6)?

<img width="457" alt="Screenshot 2024-11-19 at 14 36 24" src="https://github.com/user-attachments/assets/d41940a2-ba85-4f97-b4bd-499a4c81123b">